corehq/apps/cachehq/cachemodels.py

```diff
@@ -110,11 +110,3 @@ class UserReportsDataSourceCache(GenerationCache):
     views = [
         'userreports/data_sources_by_build_info',
     ]
-
-
-class UserReportsReportConfigCache(GenerationCache):
```

Comment:
I don't think these `GenerationCache` classes do that, the `CachedCouchDocumentMixin` superclass does. Or if you are no longer relying on these cached views (which I think we're mostly trying to move away from), you can just as easily subclass `QuickCachedDocumentMixin` to get everything else